api/graphql/schema.graphql

```diff
@@ -1203,6 +1205,11 @@ input DepartmentBelongsToMany {
   sync: [ID!]
 }
 
+input CommunityBelongsTo {
+  connect: UUID
+  disconnect: Boolean
+}
```

Comment:
I think this should be more like SkillBelongsTo, where connect is required and there's no disconnect. The search requests aren't really meant to be edited after creation, so disconnect is a bit meaningless. (We could name this to make it more specific to the talent request. That's not a pattern we've followed much, but maybe we should...)
```suggestion
input CommunityBelongsTo {
  connect: UUID!
}
```